pvr.octonet/addon.xml.in

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="pvr.octonet"
-  version="20.3.0"
+  version="21.1.0"
```

Comment:
21.0.0 as this is the first Omega version

Reply:
Done